### ISSUE TITLE
Expand on editor has empty path in callback.

### ISSUE THREAD

**Issue by lionelhorn** [josdejong/svelte-jsoneditor#180]
Hello, thanks for your editor. I've been using the previous one for quite some time.

I switched to the svelte based one and i'm trying to expand only the root object.

I've seen mention of `expand` here. https://github.com/josdejong/svelte-jsoneditor/issues/41#issuecomment-997426812

I tried to use it like this:
Path is an empty array.

```typescript
refEditor.current?.expand((path) => {
      console.log(path);
})
```

I forked your react codesandbox and changed the code for repro :  
https://codesandbox.io/s/svelte-jsoneditor-react-forked-79ique?file=/src/SvelteJSONEditor.js

The result I'm trying to achieve is root level expansion.
<img src="https://user-images.githubusercontent.com/50528276/199112666-0b5d8824-6dc7-4c0b-ae5a-6f35c830c82f.png" width=200 />

**Comment by josdejong** [comment]
Thanks for your question Lionel.

The `expand((path: string) => boolean)` method will expand all paths for which your callback function returns `true`. If you only want the root path to expand, your callback function has to return `true` when `path` equals an empty array `[]`, and `false` otherwise. This could look like:

```js
function expandRoot() {
  refEditor.current?.expand((path) => {
    return path.length === 0
  })
}
```

**Comment by lionelhorn** [comment]
Thanks Jos for the quick reply. It works properly with your example.

For some reason I was expecting a callback with a breadcrumb path like for each expandable path.

For this object for instance.
<img src="https://user-images.githubusercontent.com/50528276/199269157-28ef1875-3c71-4be3-84f6-56aa565e1f2f.png" width=200 />

I thought expand callback would be called with 
```
""
"B"
"C"
"C.D"
```

Instead it's called with
```
[]
['B']
['C']
['C', 'D']
```

**TLDR : The way it works makes it easy to expand to a specific level.**
Root only
```
 return path.length < 1;
```

Root + 1 level deep
```
 return path.length < 2;
```

**Comment by josdejong** [comment]
👍 

Yes, you can get very creative. For example the default behavior of expanding nodes on larger files is currently to expand the root object/array, and when the root is an array, expand the first item of the array:

```js
function expandMinimal(path: JSONPath) {
  return path.length === 0
    ? true // expand root
    : path.length === 1 && path[0] === '0' // first item of an array
}
```
